epoch_multinode_test.go

```diff
@@ -280,6 +291,10 @@ func (c *testComm) Broadcast(msg *Message) {
 		return
 	}
 
+	if msg.BlockMessage != nil {
```

Comment:
we convert the verified block to an unverified one so we could inject it as input to another instance.